src/app/project/board/project-tab-view/project-tab-view.component.html

```diff
@@ -0,0 +1,94 @@
+<p>Metadata for selected project is listed below.</p>
+
+<div class="properties-container">
+    <div class="properties">
+        <div *ngFor="let prop of metadata | keyvalue; let last = last; let i = index;" class="border-bottom">
+            <div *ngIf="excludeKeys.indexOf(prop.key) < 0" class="property">
+                <div class="property-label">
+                    <h3 class="label mat-subheading-1">
+                        {{ prop.key }}
+                    </h3>
+                </div>
+                <div ngSwitch="{{ prop.key }}" class="property-value">
+
+                    <!-- #################### template for discipline ################# -->
+                    <div *ngSwitchCase="'discipline'">
+                        <a href="{{ metadata['discipline']['url'] }}" target="_blank"> {{ metadata['discipline']['name'] }} </a>
+                    </div>
+
+                    <!-- #################### template for temporalCoverage ################# -->
+                    <div *ngSwitchCase="'temporalCoverage'">
+                        <a href="{{ metadata['temporalCoverage']['url'] }}" target="_blank"> {{ metadata['temporalCoverage']['name'] }} </a>
+                    </div>
+
+                    <!-- #################### template for discipline ################# -->
```

Comment:
to correct: template for URL (not discipline again I guess)

Reply:
Done.